Azure/sonic-buildimage
# Pavelsh/restart swss counters

**- What I did**
Add a functionality to restart enable_counters.service in 60 sec after swss.service restart

**- How I did it**
Add a `PartOf=` clause into enable_counters. I added 60 seconds delay to the service, because PartOf= starts working on the stop of the swss, but we need to wait until swss started.

**- How to verify it**
Build an image, install on your DUT.
After that:
`sudo reboot.`
Then wait 5 minutes and check state of counters
```
docker exec -it database redis-cli -n 2 keys "*COUNTERS:oid*" | tail
```
You should see some counters.
After that do
`systemctl restart swss` and wait a minute.
After that try 
```
docker exec -it database redis-cli -n 2 keys "*COUNTERS:oid*" | tail
```
You should see the counters.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
